Package.swift

```diff
@@ -153,10 +181,11 @@ let package = Package(
 
     .testTarget(
       name: "SwiftSyntaxTest",
-      dependencies: ["_SwiftSyntaxTestSupport", "SwiftSyntax", "SwiftSyntaxBuilder"]
+      dependencies: ["_SwiftSyntaxTestSupport", "SwiftSyntax", "SwiftSyntaxBuilder"],
+      swiftSettings: swiftSyntaxSwiftSettings
```

Comment:
Out of interest, was this causing rebuilds of the other targets?

Reply:
Nope, this PR is pure cleanup. Fixing all the rebuilds is
- https://github.com/apple/swift-format/pull/624
- https://github.com/apple/swift-format/pull/630
- https://github.com/apple/swift-stress-tester/pull/251
- https://github.com/apple/swift/pull/68562
- https://github.com/apple/swift-stress-tester/pull/252